CHANGELOG.md

```diff
@@ -0,0 +1,40 @@
+# Changelog
+
+## Unreleased (2022-03-24)
```

Comment:
Let's remove it for while and generate when we actually release it, what do you think?

Reply:
@villander works for me! Removed